Pci stages (#24748)
DataDog/documentation: layouts/shortcodes/pci-apm.md

To set up PCI compliant Application Performance Monitoring, you must meet the following requirements:

> - If not already enabled, Audit Trail is automatically enabled when the org is configured as PCI compliant. Audit Trail must be enabled and remain enabled for PCI DSS compliance.

Reviewer: ```suggestion
- Audit Trail must be enabled and remain enabled for PCI DSS compliance. If you haven't already enabled Audit Trail, it is _automatically enabled_ once the org is configured as PCI-compliant (after following the steps below).
```

- Your Datadog organization is in the US1 site.

> - All spans sent to the PCI endpoints using HTTPS only. If you are using the Agent, you should enforce HTTPS transport.

Reviewer: ```suggestion
- All spans sent to the PCI endpoints using HTTPS only. If you are using the Agent to send spans, you should enforce HTTPS transport.
```

> - **All** your endpoints will need to be changed to the PCI endpoints. 

Reviewer: ```suggestion
- **All** your endpoints need to be changed to the PCI endpoints. 
```

> - You may request access to the PCI Attestation of Compliance and Customer Responsibility Matrix on [Datadog's Trust Center][105] - note that these documents are only applicable once you have finished all the onboarding steps and have been manually configured to compliant by Datadog support.

Reviewer: ```suggestion
- You may request access to the PCI Attestation of Compliance and Customer Responsibility Matrix on [Datadog's Trust Center][105] - note that these documents are only applicable once you have finished all the onboarding steps and have been manually configured to be compliant by Datadog support.
```

To begin **onboarding**:
1. Contact [Datadog support][101] or your [Customer Success Manager][102] to request to being the PCI onboarding process while ensuring the necessary PCI requirements are met.
2. After Datadog support or Customer Success confirms that the org is PCI DSS compliant, configure the respective configuration file to send spans to the dedicated PCI compliant endpoint:
   
- `https://trace-pci.agent.datadoghq.com` for Agent and non-Agent traffic

3. For example, add the following lines to the Agent configuration file:
```
apm_config:
  apm_dd_url: <https://trace-pci.agent.datadoghq.com>
```
4. All spans that are sent to the PCI compliant endpoint(s) automatically have a set of [Sensitive Data Scanner][106] PCI rules that are applied to scrub any cardholder data. These dedicated PCI rules must be enalbed for PCI DSS compliance and are included with no additional charge.

To finish onboarding and be moved to **compliant**:
1. Inform your [Datadog support][101] or your [Customer Success Manager][102] that you have moved over **all** your endpoints to the PCI compliant endpoint(s).

> 2. Once confirmed by Datadog, your spans will be configured to be PCI compliant. Only now is your Application Performance Monitoring considered PCI compliant.

Reviewer: ```suggestion
2. Once confirmed by Datadog, your span configuration and Application Performance Monitoring is considered PCI-compliant.
```

> If you have any questions about how your now PCI compliant Application Performance Monitoring satisfies the applicable requirements under PCI DSS, contact your account manager. For PCI compliant Log Management, see [here][104].

Reviewer: ```suggestion
If you have any questions about how your now PCI-compliant Application Performance Monitoring satisfies the applicable requirements under PCI DSS, contact your account manager. See information on setting up [PCI-compliant Log Management][104].
```
